Drivers/seds/UartHwasComponent.xml

```diff
@@ -1177,6 +1182,11 @@
                 <Assignment outputVariableRef="irqNumberVar">
                   <VariableRef variableRef="uartVar.mIrqNumber" />
                 </Assignment>
+                <SendCommandPrimitive shortDescription="Subscribe to uartVar interrupt" interface="InterruptSubscriptionManagement" command="SubscribeToInterrupt">

```

Comment:
I have the feeling this is redundant.... Didn't we settle on making a single subscription?